modules/ismp/clients/polygon/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "ismp-polygon"
+version = "0.1.0"
+edition = "2021"
+description = "ISMP Consensus Client for the Polygon Heimdall+EVM Consensus Protocol"
+authors = ["Polytope Labs <hello@polytope.technology>"]
+publish = false
+
+[features]
+default = ["std"]
+std = []
```

Comment:
You need to enable std feature for all the dependencies. Look at other consensus client implementations